proc/breakpoints.go

```diff
@@ -33,9 +33,33 @@ type Breakpoint struct {
 	HitCount      map[int]uint64 // Number of times a breakpoint has been reached in a certain goroutine
 	TotalHitCount uint64         // Number of times a breakpoint has been reached
 
-	Cond ast.Expr // When Cond is not nil the breakpoint will be triggered only if evaluating Cond returns true
+	// When DeferCond is set the breakpoint will only trigger
+	// if the caller is runtime.gopanic or if the return address
+	// is in the DeferReturns array.
+	// Next sets DeferCond for the breakpoint it sets on the
+	// deferred function, DeferReturns is populated with the
+	// addresses of calls to runtime.deferreturn in the current
+	// function. This insures that the breakpoint on the deferred
+	// function only triggers on panic or on the defer call to
+	// the function, not when the function is called directly
+	DeferCond    bool
+	DeferReturns []uint64
+	Cond         ast.Expr // When Cond is not nil the breakpoint will be triggered only if evaluating Cond returns true
```

Comment:
nit: Move this comment above the member, and drop the initial `When`, and say something like `Cond, if not nil, ...`